miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -506,6 +506,9 @@ private void applyChanges(Sample target, Sample source) throws IOException {
     target.setDescription(source.getDescription());
     target.setEmpty(source.isEmpty());
     target.setVolume(source.getVolume());
+    if (!isStringBlankOrNull(source.getIdentificationBarcode())) {
```

Comment:
Wrong barcode was added, correct one is unknown. Barcode accidentally added to ghost sample.

I think you should be able to remove it. If not, then the UI needs to prevent it too and show an error when you try to clear it